python/run_workflow.py

```diff
@@ -8,12 +9,15 @@ async def main():
     # Create client connected to server at the given address
     client = await Client.connect("localhost:7233")
 
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--parallel', '--parallel-compensations', '-p', action='store_true')
+    args = parser.parse_args()
     # Execute a workflow
-    result = await client.execute_workflow(
-        BreakfastWorkflow.run, id="breakfast-workflow", task_queue="breakfast-queue"
+    await client.execute_workflow(
+        BreakfastWorkflow.run, args.parallel, id="breakfast-workflow", task_queue="breakfast-queue"
     )
 
-    print(f"Result: {result}")
+    print("Started workflow")
```

Comment:
Actually, you completed the workflow at this point :-) (`start_workflow` is a different call)

Reply:
ahem, yes. let's pretend I never had this here. 🤦